Update to 3.32

mvainola opened the pull request.

Erick555 commented on org.gnome.Evince.json:
```diff
@@ -53,7 +53,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/uclouvain/openjpeg.git",
-                    "commit": "d2205ba2ee78faeea659263383446c4472b1f9df"
```
openjpg is avalaible in gnome runtime. I think you can remove this module completely

mvainola replied:
My local build succeeded without the module, I'll remove it.

hadess replied:
This should be accompanied with a `-DENABLE_LIBOPENJPEG=...` in the poppler module's configure options, to make sure that JPEG2000 images support isn't silently disabled.

mvainola commented:
I added a patch for the missing release tag in the AppData file which should allow it to pass validation. I'll create an issue about it upstream.

hadess commented:
The appdata change should be done upstream, as you mentioned. Could you please also split up the various library upgrades into their own commits?

jascrain commented:
Several of these things should also be done upstream. The evince repo has a "development" flatpak manifest which should be kept in sync.

jascrain commented on org.gnome.Evince.json:
```diff
@@ -99,8 +82,8 @@
                     "type": "archive",
-                    "url": "https://poppler.freedesktop.org/poppler-0.74.0.tar.xz",
-                    "sha256": "92e09fd3302567fd36146b36bb707db43ce436e8841219025a82ea9fb0076b2f"
+                    "url": "https://poppler.freedesktop.org/poppler-0.77.0.tar.xz",
```
Poppler has releases about once per month, so this dependency is already out of date.

```suggestion
                    "url": "https://poppler.freedesktop.org/poppler-0.79.0.tar.xz",
```

jascrain commented on org.gnome.Evince.json:
```diff
@@ -99,8 +82,8 @@
-                    "url": "https://poppler.freedesktop.org/poppler-0.74.0.tar.xz",
-                    "sha256": "92e09fd3302567fd36146b36bb707db43ce436e8841219025a82ea9fb0076b2f"
+                    "url": "https://poppler.freedesktop.org/poppler-0.77.0.tar.xz",
+                    "sha256": "7267eb4cbccd64a58244b8211603c1c1b6bf32c7f6a4ced2642865346102f36b"
```
```suggestion
                    "sha256": "f985a4608fe592d2546d9d37d4182e502ff6b4c42f8db4be0a021a1c369528c8"
```

mvainola commented:
I split all the updates into individual commits. I added `-DENABLE_LIBOPENJPEG=openjpeg2` to Poppler's config-opts as suggested by @hadess and added gnome-desktop as well as other changes found in @jascrain's [upstream manifest update](https://gitlab.gnome.org/GNOME/evince/merge_requests/178).

I also opened a [merge request for the AppData change](https://gitlab.gnome.org/GNOME/evince/merge_requests/179).
